09-sorting-algorithms/lib/quicksort.js

```diff
@@ -0,0 +1,30 @@
+'use strict'
+
+let quicksort = (array) => {
+  if (!array) {
+    throw new Error('ERROR: you must pass in an array');
+  }
+  if (array.length <= 1) {
+    return array;
+  } else {
+    let pivot = array.pop();
+    let left = [];
+    let right = [];
+    let newArray = [];
+    let len = array.length;
+
+    for (let i = 0; i < len; i++) {
```

Comment:
You could have used a forEach() loop here to accomplish the same thing. (It's more declarative.)